src/patternfly/components/OverflowMenu/overflow-menu.scss

```diff
@@ -17,6 +18,21 @@
 
   display: inline-flex;
   align-items: center;
+
+  // set this var here so specificity is 20
+  .pf-c-divider {
```

Comment:
Same note here re: scope - it's fair to assume a user or we might introduce a button or split button or something with a divider and these styles would conflict.